### PR TITLE
[#31080] Add large button

### PR DESCRIPTION
**Pull request by bienvu**
Add large button variant
https://agov_pr526.agov.com.au/styleguide/agov_whitlam/section-components.html#kssref-components-button

### CODE REVIEW

**Comment by joannguyen** [comment]
Hi @JohnAlbin, Do we need to show all kinds of button in large OR only need to show one large button like a link button?

File: themes/agov/agov_whitlam/sass/components/button/_button.scss [hunk 1]
```diff
@@ -12,6 +12,7 @@
 // Markup: button.hbs
 //
 // :hover - Hover styling.
+// button--large - Large button styling.
```
Comment (JohnAlbin) [inline themes/agov/agov_whitlam/sass/components/button/_button.scss]: You are missing a `.` before the class name. e.g.

```
// .button--large - Large button styling.
```

**Comment by JohnAlbin** [comment]
> Do we need to show all kinds of button in large OR only need to show one large button like a link button?

If you fix the docs for the modifier (I commented above), it should show a large button for all kinds of button types.

**Comment by bienvu** [comment]
Updated

**Comment by kimpepper** [comment]
Feedback has been resolved.

**Pull request merged**